docs/examples/101/data-factory-v2-blob-to-blob-copy/main.json

```diff
@@ -124,47 +124,7 @@
       "name": "[format('{0}/{1}', variables('dataFactoryName'), variables('pipelineName'))]",
       "properties": {
         "activities": [
-          {
-            "name": "MyCopyActivity",
-            "type": "Copy",
-            "policy": {
-              "timeout": "7.00:00:00",
-              "retry": 0,
-              "retryIntervalInSeconds": 30,
-              "secureOutput": false,
-              "secureInput": false
-            },
-            "typeProperties": {
-              "source": {
-                "type": "BinarySource",
-                "storeSettings": {
-                  "type": "AzureBlobStorageReadSettings",
-                  "recursive": true
-                }
-              },
-              "sink": {
-                "type": "BinarySink",
-                "storeSettings": {
-                  "type": "AzureBlobStorageWriterSettings"
-                }
-              },
-              "enableStaging": false
-            },
-            "inputs": [
-              {
-                "referenceName": "[variables('DataFactoryDataSetInName')]",
-                "type": "DatasetReference",
-                "properties": {}
-              }
-            ],
-            "outputs": [
-              {
-                "referenceName": "[variables('DataFactoryDataSetOutName')]",
-                "type": "DatasetReference",
-                "properties": {}
-              }
-            ]
-          }
+          "[createObject('name', 'MyCopyActivity', 'type', 'Copy', 'policy', createObject('timeout', '7.00:00:00', 'retry', 0, 'retryIntervalInSeconds', 30, 'secureOutput', false(), 'secureInput', false()), 'typeProperties', createObject('source', createObject('type', 'BinarySource', 'storeSettings', createObject('type', 'AzureBlobStorageReadSettings', 'recursive', true())), 'sink', createObject('type', 'BinarySink', 'storeSettings', createObject('type', 'AzureBlobStorageWriterSettings')), 'enableStaging', false()), 'inputs', createArray(createObject('referenceName', variables('DataFactoryDataSetInName'), 'type', 'DatasetReference', 'properties', createObject())), 'outputs', createArray(createObject('referenceName', variables('DataFactoryDataSetOutName'), 'type', 'DatasetReference', 'properties', createObject())))]"
```

Comment:
>createObject [](http://example.com/codeflow?start=12&length=12)

Why did this change? There's no corresponding change to the source Bicep file. I see the same issue in a few more examples as well.

Reply:
I suspect some of the codegen changes caused this.

Reply:
Ok, I think the problem is this change: https://github.com/Azure/bicep/pull/2501/files/bf28e4ba2ebf8ba6c4718a2af425667569911be9#diff-04c0c0e943c27e35f8e88ed7a9930cf43803e53b105a7643d4fe6e129987ff62L140

I merged it with the `any` detection in the ExpressionConverter to have a single place where special functions have their logic, even `any` - to decouple a bit relation between Symbols part and Emitter part.